src/main/res/values/strings.xml

```diff
@@ -494,7 +494,7 @@ limitations under the License.
     <string name="track_delete_one_confirm_title">Delete track?</string>
     <string name="track_delete_multiple_confirm_message">The selected tracks and their markers will be permanently deleted from the device.</string>
     <string name="track_delete_progress_message">Deleting&#8230;</string>
-    <string name="track_delete_progress">%1$d/%2$d</string>
```

Comment:
need to be removed in the other translations as well.

Reply:
Ok, I thought this will be handled by weblate later on. No problem, I'll remove them.

Reply:
It triggers linter issues; a while ago this could be done in AndroidStudio (with this translation UI), but at least that stopped working a while ago...